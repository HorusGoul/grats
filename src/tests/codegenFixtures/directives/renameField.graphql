directive @methodName(name: String!) on FIELD_DEFINITION

type Query {
  greeting: String @methodName(name: "salutation")
}
