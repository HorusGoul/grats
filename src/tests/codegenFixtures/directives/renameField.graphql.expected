-----------------
INPUT
----------------- 
directive @methodName(name: String!) on FIELD_DEFINITION

type Query {
  greeting: String @methodName(name: "salutation")
}

-----------------
OUTPUT
-----------------
import { GraphQLSchema, GraphQLObjectType, GraphQLString } from "graphql";
const QueryType: GraphQLObjectType = new GraphQLObjectType({
    name: "Query",
    fields() {
        return {
            greeting: {
                name: "greeting",
                type: GraphQLString
            }
        };
    }
});
const schema = new GraphQLSchema({
    query: QueryType
});
export { schema };
